apps/crn-server/src/queries/events.queries.ts

```diff
@@ -58,6 +58,7 @@ export const eventContentFragment = gql`
             id
             flatData {
               alumniSinceDate
+              alumniLocation
```

Comment:
I believe we don't need this `alumniLocation` everywhere. We do need this in `user` because it appears in the user profile page
![user-location](https://user-images.githubusercontent.com/16595804/216300222-23c41235-8522-486b-a3bc-b3cb6f8f90b1.png)


but in the event, for example, we need the `alumniSinceDate` to flag with this "A" badge beside the name of the user, but we don't show the `alumniLocation`

![event](https://user-images.githubusercontent.com/16595804/216300544-05da5011-fb12-4ade-bc99-446fdbecaacc.png)

I haven't took a look at groups and research output in the hub, but I believe it's the same thing